# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsContentSink.h
# Commit: 2fe2afdee09d
# Full Hash: 2fe2afdee09dd5d18d7208f5b4186e659ebdde7e
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-02 21:26:33
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 11: Don't preload anchored links from early hints r=necko-reviewers,valentin
#   
#   They won't get passed to the PreloadService anyway. Resulting in an unnecessary
#   additional load
#   
# ==============================================================================

diff -r 8bdb1f682d22 -r 2fe2afdee09d dom/base/nsContentSink.h
--- a/dom/base/nsContentSink.h	Fri Dec 02 09:45:27 2022 +0000
+++ b/dom/base/nsContentSink.h	Fri Dec 02 09:45:28 2022 +0000
@@ -115,7 +115,6 @@
   virtual void UpdateChildCounts() = 0;
 
   bool IsTimeToNotify();
-  bool LinkContextIsOurDocument(const nsAString& aAnchor);
 
  protected:
   nsContentSink();